JS-vote/script.js

```diff
@@ -0,0 +1,8 @@
+const personAge = 32;
+if (personAge >= 18) {
+    citizenCanVote = true;
```

Comment:
Why did you set this undeclared variable here? Was it part of the exercise requirement?